docs/source/release/v6.10.0/Framework/Python/Bugfixes/36073.rst

```diff
@@ -0,0 +1,3 @@
+- Input validation errors have been corrected for numerical line edits of the Filter Events interface.
+- Fix crash occurring when incorrect `TOF Correction To Sample` value is selected for the input workspace. Now error message is displayed instead of crash.
```

Comment:
```suggestion
- Fix crash occurring when an incorrect `TOF Correction To Sample` value is selected for the input workspace. Now error message is displayed instead of crash.
```

Reply:
@adriazalvarez I know I am nit-picking here!

Reply:
@SilkeSchomann No problem!